pkg/cmd/admin/registry/registry.go

```diff
@@ -294,6 +302,22 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 			},
 		}
 		objects = app.AddServices(objects)
+
+		if generateServiceAccount {
+			// Add the new service account to "privileged"
+			scc, err := kClient.SecurityContextConstraints().Get("privileged")
+			if err != nil {
+				return fmt.Errorf("can't check for existing security context constraints privileged: %v", err)
```

Comment:
Can we get a bit more context in this error message? I could see myself running this and thinking "I just wanted to make a registry, why is it complaining about a security context constraint?" :-)

Maybe something like "Unable to create the registry service account: can't check for ..."